### PR TITLE
Adding forgotten 1.3 features

### PR DESCRIPTION
**Pull request by lzap**
I overlooked puppet modules this time, sorry about that. All features/bugs
were taken directly from git, except puppet modules. I hope this is all, if
not please ping me.

### CODE REVIEW

File: _includes/manuals/1.3/1.2_release_notes.md [hunk 1]
```diff
@@ -263,6 +263,8 @@ manual section.
 - Method delete_all is used for performance reasons when orphaning records ([#3202](http://projects.theforeman.org/issues/3202))
 - Added plugins repo to foreman-release ([#2981](http://projects.theforeman.org/issues/2981))
 - Command passenger-status is now working ([#3197](http://projects.theforeman.org/issues/3197))
+- TFTP module now supports Amazon Linux
```
Comment (ekohl) [inline _includes/manuals/1.3/1.2_release_notes.md]: This should be the other way around: all modules but TFTP support Amazon Linux. https://github.com/theforeman/puppet-tftp/pull/14 should fix it and we should rectify this in 1.3.1 imho.

File: _includes/manuals/1.3/1.2_release_notes.md [hunk 2]
```diff
@@ -263,6 +263,8 @@ manual section.
...
+- Full BMC support
```
Comment (ekohl) [inline _includes/manuals/1.3/1.2_release_notes.md]: Could link to http://theforeman.org/issues/2853.

**Comment by lzap** [comment]
Done, thanks!

File: _includes/manuals/1.3/1.2_release_notes.md [hunk 3]
```diff
@@ -263,6 +263,8 @@ manual section.
 - Method delete_all is used for performance reasons when orphaning records ([#3202](http://projects.theforeman.org/issues/3202))
 - Added plugins repo to foreman-release ([#2981](http://projects.theforeman.org/issues/2981))
 - Command passenger-status is now working ([#3197](http://projects.theforeman.org/issues/3197))
+- Puppet modules now support Amazon Linux ([##14](https://github.com/theforeman/puppet-tftp/pull/14)
```
Comment (ekohl) [inline _includes/manuals/1.3/1.2_release_notes.md]: This implies there is full Amazon support, but TFTP has yet to be merged and depends on support in puppetlabs-xinetd as well. For puppet-dns and puppet-dhcp there is no Amazon support at all nor any issue/PR open and will `fail()`.

Ones that were merged:
- https://github.com/theforeman/puppet-passenger/pull/17
- https://github.com/theforeman/puppet-foreman/pull/113

**Comment by lzap** [comment]
Thanks :-)

**Comment by ekohl** [comment]
:+1:

**Pull request merged**